src/lib/I18n/index.js

```diff
@@ -38,21 +38,26 @@ export default class I18n {
     this._loadLocale = loadLocale;
     this._cache = {};
     RUNTIME.instances.add(this);
-    this._load(DEFAULT_LOCALE);
-    this._load(RUNTIME.locale);
+    this.pInitiate = Promise.all([this._load(DEFAULT_LOCALE), this._load(RUNTIME.locale)]);
+    this.initiated = false;
```

Comment:
I would also recommend making this a this._initiated + getter pair so that it is clear that this value should not be set externally.

Reply:
I want it to be an public property, given the instance more flexibility